src/main/java/com/zollingpaper/backend/board/controller/BoardController.java

```diff
@@ -0,0 +1,38 @@
+package com.zollingpaper.backend.board.controller;
+
+import com.zollingpaper.backend.board.dto.BoardDetailResponse;
+import com.zollingpaper.backend.board.dto.BoardSaveRequest;
+import com.zollingpaper.backend.board.service.BoardService;
+import java.net.URI;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BoardController {
+
+    private final BoardService boardService;
+
+    public BoardController(BoardService boardService) {
+        this.boardService = boardService;
+    }
+
+    @PostMapping("/board")
+    public ResponseEntity<Void> saveBoard(
+            @RequestBody BoardSaveRequest request
+    ) {
+        Long boardId = boardService.saveBoard(request);
+        return ResponseEntity.created(URI.create("/board/" + boardId.toString())).build();
```

Comment:
```suggestion
        Long boardId = boardService.saveBoard(request);
        return ResponseEntity.created(URI.create("/board/" + boardId)).build();
```

`toString()` 없이도 가능한데 사용한 이유가 있을까요?

Reply:
이전 명세는 UUID를 사용하는 것이었는데, 해당 구현 과정에서 작성해두고 수정하지 않은 것 같습니다. toString은 제거하도록 하겠습니다. 감사합니다!